airflow/www/views.py

```diff
@@ -3294,7 +3292,7 @@ def next_run_datasets(self, dag_id):
             latest_run = dag_model.get_last_dagrun(session=session)
 
             events = [
-                dict(info)
+                dict(info._mapping)
```

Comment:
I assume this attribute is officially supported although it looks private

Reply:
Yeah... That rare case when access to properties with leading underscore is official/recommended way:
- https://docs.sqlalchemy.org/en/14/core/connections.html#sqlalchemy.engine.Row._mapping
- https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Row._mapping

This one is come from the warning it self
 
```console
The Row.keys() method is considered legacy as of the 1.x series of SQLAlchemy and will be removed in 2.0.
Use the namedtuple standard accessor Row._fields, or for full mapping behavior use row._mapping.keys()
(Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
```